metrics/metrics-integration/metrics-prometheus/pom.xml

```diff
@@ -43,7 +42,10 @@
       <groupId>org.apache.servicecomb</groupId>
       <artifactId>metrics-core</artifactId>
     </dependency>
-
+    <dependency>
```

Comment:
Shouldn't the scope set to 'test' here?

Reply:
why test?
all servicecomb measurement come from metrics-core.

Reply:
I mean 'foundation-test-scaffolding'